pages/sites/[slug]/index.tsx

```diff
@@ -90,13 +117,17 @@ export default function Donate({
         setCurrencyCode(currency);
         setInternalLanguage(i18n.language);
         try {
-          const projects = await getRequest<MapProject[]>(`/app/projects`, {
-            _scope: 'map',
-            currency: currency,
-            tenant: TENANT_ID,
-            'filter[purpose]': 'trees,conservation',
-            locale: i18n.language,
-          });
+          const projects = await getRequest<MapProject[]>(
+            pageProps.tenantConfig.id,
+            `/app/projects`,
+            {
+              _scope: 'map',
+              currency: currency,
+              tenant: pageProps.tenantConfig.id,
+              'filter[purpose]': 'trees,conservation',
+              locale: i18n.language,
+            }
+          );
```

Comment:
@Shreyaschorge How come we need to pass tenantConfig.id twice into this `GetRequest` call?

Reply:
In some places, the 'tenant-key' is passed in the query parameter object, while in others, it is not. I have removed the 'tenant-key' from here.

Reply:
For now we'll send the tenant both in header and the query params. can be considered in #1976